src/centrality/betweenness.c

```diff
@@ -1018,3 +1015,841 @@ int igraph_edge_betweenness_estimate(const igraph_t *graph, igraph_vector_t *res
+     * i.e. its precision is lower, it is effectively immune to overflow. The impact on the precision
+     * of the final result is negligible. The max betweenness is correct to 14 decimal digits,
+     * i.e. the precision limit of 'double', even for a 101*101 grid graph. */
+    double *nrgeo = 0;
+    double *tmpscore;
+    igraph_stack_t stack = IGRAPH_STACK_NULL;
+    long int source;
+    long int j, nneis;
+    igraph_vector_int_t *neis;
+    igraph_vit_t vit;
+    unsigned char *is_target;
+    igraph_adjlist_t adjlist_out, adjlist_in;
+    igraph_adjlist_t *adjlist_out_p, *adjlist_in_p;
+
+    if (weights) {
+        return igraph_i_betweenness_subset_weighted(graph, res, vids, directed,
+                                                    weights);
+    }
+
+    if (!igraph_vs_is_all(&vids)) {
+        /* subset */
+        IGRAPH_CHECK(igraph_vit_create(graph, vids, &vit));
+        IGRAPH_FINALLY(igraph_vit_destroy, &vit); 
+        to_reach = IGRAPH_VIT_SIZE(vit);
+        igraph_vector_resize(res, no_of_nodes);
+        igraph_vector_fill(res, 0);
+    } else {
+        /* only  */
+        return igraph_betweenness_cutoff(graph, res, vids, directed,
+                                                    weights, -1);
+    }
+
+    is_target = igraph_Calloc(no_of_nodes, unsigned char);
+    if (is_target == 0) {
+        IGRAPH_ERROR("Can't calculate shortest paths", IGRAPH_ENOMEM);
+    }
+    IGRAPH_FINALLY(igraph_free, is_target); 
+
+    for (IGRAPH_VIT_RESET(vit); !IGRAPH_VIT_END(vit); IGRAPH_VIT_NEXT(vit)){
+        if (!is_target[(long int) IGRAPH_VIT_GET(vit)]){
+            is_target[(long int) IGRAPH_VIT_GET(vit)] = 1;
+        }
+        else{
+            to_reach --;
+        }
+    }
+
+    directed = directed && igraph_is_directed(graph);
+    if (directed) {
+        IGRAPH_CHECK(igraph_adjlist_init(graph, &adjlist_out, IGRAPH_OUT, IGRAPH_LOOPS_ONCE, IGRAPH_MULTIPLE));
+        IGRAPH_FINALLY(igraph_adjlist_destroy, &adjlist_out);
+        IGRAPH_CHECK(igraph_adjlist_init(graph, &adjlist_in, IGRAPH_IN, IGRAPH_LOOPS_ONCE, IGRAPH_MULTIPLE));
+        IGRAPH_FINALLY(igraph_adjlist_destroy, &adjlist_in);
+        adjlist_out_p = &adjlist_out;
+        adjlist_in_p = &adjlist_in;
+    } else {
+        IGRAPH_CHECK(igraph_adjlist_init(graph, &adjlist_out, IGRAPH_ALL, IGRAPH_LOOPS_TWICE, IGRAPH_MULTIPLE));
+        IGRAPH_FINALLY(igraph_adjlist_destroy, &adjlist_out);
+        IGRAPH_CHECK(igraph_adjlist_init(graph, &adjlist_in, IGRAPH_ALL, IGRAPH_LOOPS_TWICE, IGRAPH_MULTIPLE));
+        IGRAPH_FINALLY(igraph_adjlist_destroy, &adjlist_in);
+        adjlist_out_p = &adjlist_out;
+        adjlist_in_p = &adjlist_in;
+    }
+    for (j = 0; j < no_of_nodes; j++) {
+        igraph_vector_int_clear(igraph_adjlist_get(adjlist_in_p, j));
+    }
+
+    distance = igraph_Calloc(no_of_nodes, long int);
+    if (distance == 0) {
+        IGRAPH_ERROR("betweenness failed", IGRAPH_ENOMEM);
```

Comment:
It seems that these messages were copied from an old version of the standard betweenness code. Please use the new message, and refer to the [error reporting guidelines](https://github.com/igraph/igraph/wiki/Error-reporting-guidelines).